src/refactorings/inline/inline-variable/find-inlinable-code.ts

```diff
@@ -248,7 +249,7 @@ class InlinableIdentifier implements InlinableCode {
         let selection = Selection.fromAST(loc);
 
         if (t.isTemplateLiteral(parent)) {
-          code = code.replace(/^("|'|\`)/, "").replace(/("|'|\`)$/, "");
```

Comment:
That broke the tests. It should really be `

Was it changed automatically? If so, we should fix some config.